bipm/ontology/.htaccess

```diff
@@ -0,0 +1,2 @@
+RewriteEngine On
+RewriteRule ^ontology$ https://raw.githubusercontent.com/kongyingying123/bipm-ontology/main/bipm-ontology.owl [R=303,L]
```

Comment:
you are minting `https://w3id.org/bipm/ontology/ontology` Is this what you intend?
I suspect you want to edit this to be 
```
RewriteRule ^$ https://raw.githubusercontent.com/kongyingying123/bipm-ontology/main/bipm-ontology.owl [R=303,L]
```
so you mint `https://w3id.org/bipm/ontology/`